integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -488,7 +490,24 @@ case class CarbonLoadDataCommand(
         }
         InternalRow.fromSeq(data)
       }
-      LogicalRDD(attributes, rdd)(sparkSession)
+      if (updateModel.isDefined) {
+        sparkSession.sparkContext.setLocalProperty(EXECUTION_ID_KEY, null)
+        // In case of update, we don't need the segmrntid column in case of partitioning
+        val dropAttributes = attributes.dropRight(1)
+        val finalOutput = relation.output.map { attr =>
+          dropAttributes.find { d =>
+            val index = d.name.lastIndexOf("-updatedColumn")
```

Comment:
find a better way to get the columns in order from update flow instead of doing string manipulations

Reply:
It requires change the order in UpdateCommand, so it impacts the actual flow of IUD. So I guess better handle here